libstuff/libstuff.h

```diff
@@ -230,14 +230,18 @@ void SSyslogSocketDirect(int priority, const char* format, ...);
 // Atomic pointer to the syslog function that we'll actually use. Easy to change to `syslog` or `SSyslogSocketDirect`.
 extern atomic<void (*)(int priority, const char *format, ...)> SSyslogFunc;
 
+//template <typename T>
+//string addLogParams(const string& message, const map<string, T> params);
```

Comment:
If you are onboard with this PR, I will update some logs in Auth, and will try to make it work with maps that also contain ints as values.

Reply:
Actually, I realized it is not possible to achieve what I wanted to achieve. We'll have to always pass a `map<string, string>`